README.md

```diff
@@ -35,225 +48,12 @@ Be sure to check out [Bubbles][bubbles], a library of common UI components for B
 
 * * *
```

Comment:
We can drop this divider now that the massive tutorial has been moved elsewhere.